packages/tools/generator-widget/generators/app/index.js

```diff
@@ -166,10 +166,14 @@ class MxGenerator extends Generator {
     }
 
     _writeUtilityFiles() {
-        this._copyTemplate("commons/_gitignore", ".gitignore");
+        this._copyTemplate("commons/_gitignore", ".gitignore", {
+            projectIgnorePath: this.widget.projectPath.replace(/\.\//g, "")
+        });
         this._copyFile(`commons/eslintrc.${this.widget.isLanguageTS ? "ts" : "js"}.js`, ".eslintrc.js");
         this._copyFile("commons/prettier.config.js", "prettier.config.js");
-        this._copyTemplate("commons/.prettierignore", ".prettierignore");
+        this._copyTemplate("commons/.prettierignore", ".prettierignore", {
```

Comment:
Is this one also being copied?

Reply:
Yes, I checked. It was only `.gitignore` that was affected.